dev/provider_packages/prepare_provider_packages.py

```diff
@@ -54,6 +54,10 @@
 
 ALL_PYTHON_VERSIONS = ["3.7", "3.8", "3.9", "3.10"]
 
+MIN_AIRFLOW_VERSION = "2.4.0"
+
+MIN_AIRFLOW_VERSION_EXCEPTIONS = {"openlineage": "2.6.0"}
```

Comment:
cc: @eladkal Here we specify min airflow version and those packages that exceptionally have different min version set.

Reply:
That's why I put it very closely to the MIN_AIRFLOW_VERSION - it's less likely to forget - because the only time you modify it will be when you bump the MIN_AIRFLOW_VERSION. I have no other idea how to make it "less" likely to forget. I think in real life there is always something we have to do consciously and verify the result  :)

Honestly - there is no better way I think.

Reply:
maybe we could also add a comment just above or below MIN_AIRFLOW_VERSION mentioning how exceptions can be set with MIN_AIRFLOW_VERSION_EXCEPTIONS. Or is it too much? :)

Reply:
No need... This is script for release manager.
We know what it does and how.
The issue was more about keep it sync with the provider updates (if choose to update min airflow version outside of regular cycle) but that is not issue worst case we will discover it when preparing release and it can be adressed then.

Reply:
I can add it though :)....